avocado/core/jobdata.py

```diff
@@ -173,7 +170,7 @@ def get_resultsdir(logdir, jobid):
                 from logging import getLogger
                 getLogger("avocado.app").error("hash '%s' is not unique "
                                                "enough", jobid)
-                sys.exit(exit_codes.AVOCADO_JOB_FAIL)
```

Comment:
Well actually it'd be even better to avoid the logger and simply raise `ValueError("hash ...")` and log it inside the plugins code. Imagine there is a bug in jobdata and you get ValueError on a different place. You'd have no note about the issue.